atomicapp/cli/main.py

```diff
@@ -51,7 +51,7 @@ def print_app_location(app_path):
 def cli_genanswers(args):
     argdict = args.__dict__
     nm = NuleculeManager(app_spec=argdict['app_spec'],
-                         destination='none')
+                         destination='none', cli=argdict)
```

Comment:
Errrr...

argdict is already passed in `nm.genanswers(**argdict)`

Why do we need to pass it again? Could you change it so we simply pass it via NuleculeManager and remove the need to pass it within `nm.genanswers(**argdict)`?